### DISCUSSION TITLE
Some Agents do not get Text Formatting options in the Reply Box - is it a Bug ?

### DISCUSSION THREAD

**Discussion by devrulabs** [chatwoot/chatwoot#2484] [Q&A]
Agents do not get the Text Formatting option in the reply Box and also Enter to Send checkbox is missing:

![image](https://user-images.githubusercontent.com/82273807/122865685-59f96a00-d359-11eb-84fc-314125f31ed3.png)

However the same is Available in the Private Message section :

![image](https://user-images.githubusercontent.com/82273807/122865811-9036e980-d359-11eb-8d9a-9a36096f7825.png)

Surprisingly one of the user is able to see the text formatting option 

![image](https://user-images.githubusercontent.com/82273807/122866090-fd4a7f00-d359-11eb-8c4d-233ba4bbac77.png)

**Comment by devrulabs** [comment]
Can see these settings are stored in the user table as ui_settings, how to change these settings as an Admin ?

**Reply by pranavrajs** [reply]
@devrulabs Which channel is this? Some of the channels do not support rich content messages.

**Reply by devrulabs** [reply]
@pranavrajs this is for API Channel

**Reply by sojan-official** [reply]
@devrulabs We haven't enabled the rich text editor for the API channels. We will take it up in future releases.
